packages/relay/tests/assertions.ts

```diff
@@ -1,11 +1,34 @@
-import { expect } from 'chai';
+/*
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import chai, { expect } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
 import { JsonRpcError } from '../src';
 import { EthImpl } from '../src/lib/eth';
 import { Block, Transaction } from '../src/lib/model';
 import { numberTo0x } from '../src/formatters';
 import constants from '../src/lib/constants';
 import { BASE_FEE_PER_GAS_DEFAULT } from './lib/eth/eth-config';
 
+chai.use(chaiAsPromised);
```

Comment:
Yes, there are utility functions which use `expect(...).eventually` inside, but `assertions.ts` was previously relying on other spec files which import it to have called `chai.use(chaiAsPromised)` themselves, instead of doing it here. 

I think they were extracted into this helper file at some point, but the person who did it forgot to call `chai.use(chaiAsPromised)` after extracting it into a separate file.